samples/android-app/build.gradle.kts

```diff
@@ -26,6 +26,12 @@ configure<AppExtension> {
         }
     }
 
+    flavorDimensions("beer")
+    productFlavors {
+        register("weissbier")
+        register("kellerbier")
+    }
```

Comment:
Can you explain this?

Reply:
It is android build variants. 

For example, in this case, app can include special sources/resources that only belongs to `"weissbier"` variant on top of main sources/resources.

Such approach allows to have multiple variants of the same app, like special variant for play store and variant of the app for F-Droid store.

Reply:
Cool, good to know!